utils/parametric/_library_client.py

```diff
@@ -25,6 +25,7 @@ class StartSpanResponse(TypedDict):
 class Link(TypedDict):
     parent_id: int  # 0 to extract from headers
     attributes: dict
+    http_headers: List[Tuple[str, str]]
```

Comment:
Align `Link` object sent by the test client with the grpc endpoint: https://github.com/DataDog/system-tests/blob/ed594c69dcbe212b12db2a44ad0c6302f9f4d690/utils/parametric/protos/apm_test_client.proto#L51